File: hw/cevallos/u2/hw19FarmSystem/Python/Cage.py
class Cage:
    def __init__(self, id, description, type, location):
        self.id = id
        self.description = description
        self.type = type  # 1=coop, 2=table, 3=pens
        self.location = location
    
    def __str__(self):
        return f"Cage{{id={self.id}, description={self.description}, type={self.type}, location={self.location}}}"

File: hw/cevallos/u2/hw19FarmSystem/JavaScript/Cage.js
class Cage {
    constructor(id, description, type, location) {
        this.id = id;
        this.description = description;
        this.type = type; // 1=coop, 2=table, 3=pens
        this.location = location;
    }

    toString() {
        return `Cage{id=${this.id}, description=${this.description}, type=${this.type}, location=${this.location}}`;
    }
}

module.exports = Cage;